### DISCUSSION TITLE
print is erro : content  is overlap

### DISCUSSION THREAD

**Discussion by goodgame365** [CoderLine/alphaTab#807] [General]
like this:
![1](https://user-images.githubusercontent.com/17884664/160675603-532198ed-8eb8-4aa4-ba2c-dd2a72f3cc41.jpg)

alse  happen on official web :https://www.alphatab.net 

This problem exists after this  version 1.2 But The following versions 1.1 are normal

**Comment by Danielku15** [comment]
Covered by https://github.com/CoderLine/alphaTab/issues/810 (as you reported it there too). Should be fixed soon.
